content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,210 @@
+
+- `repo` - specifies the location of the DVC project. It can be a URL or a file
+  system path. Both HTTP and SSH protocols are supported for online Git repos
+  (e.g. `[user@]server:project.git`). _Default_: The current project (found by
+  walking up from the current working directory tree).
+
+- `stages` - one or more names of the stage(s) to retrieve params from.
+  _Default_: `None` (all parameters from all stages will be retrieved).
+
+- `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
+  a branch or tag name, a commit hash or an
+  [experiment](/doc/command-reference/exp) name). If `repo` is not a Git repo,
+  this option is ignored. _Default_: `None` (current working tree will be used)
+
+- `deps` - whether to retrieve only params that are stage dependencies. Accepts
+  `True` or `False` (_default_).
+
+## Example: Filter by parameter file(s)s
+
+> Working on
+> https://github.com/iterative/pipeline-conifguration/tree/main/multi-params-files
+
+You can pass a single param file path as target:
+
+```py
+import json
+import dvc.api
+params = dvc.api.params_show("params.yaml")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "run_mode": "prod",
+  "configs": {
+    "dev": "configs/params_dev.yaml",
+    ...
+  },
+  "evaluate": {
+    "dataset": "micro",
+    ...
+```
+
+Or multiple targets:
+
+```py
+import json
+import dvc.api
+params = dvc.api.params_show(
+  "configs/params_dev.yaml", "configs/params_prod.yaml")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "configs/params_prod.yaml:run_mode": "prod",
+  "configs/params_prod.yaml:config_file": "configs/params_prod.yaml",
+  "configs/params_prod.yaml:data_load": {
+    ...
+  "configs/params_dev.yaml:run_mode": "dev",
+  "configs/params_dev.yaml:config_file": "configs/params_dev.yaml",
+  "configs/params_dev.yaml:data_load": {
+    ...
+```
+
+## Example: Specify a project version
+
+> Working on
+> https://github.com/iterative/example-get-started/tree/tune-hyperparams
+
+You can retrieve params from arbitrary Git commits, for example a branch name:
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(rev="tune-hyperparams")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "prepare": {
+    "split": 0.2,
+    "seed": 20170428
+  },
+  "featurize": {
+    "max_features": 200,
+    "ngrams": 2
+  },
+  "train": {
+    "seed": 20170428,
+    "n_est": 100,
+    "min_split": 8
+  }
+}
+```
+
+## Example: Use a remote DVC repository
+
+You can use the `repo` argument to retrieve parameters from any <abbr>DVC
+repository</abbr> without having to clone it locally.
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(
+    repo="https://github.com/iterative/demo-fashion-mnist")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "train": {
+    "batch_size": 128,
+    "hidden_units": 64,
+    "dropout": 0.4,
+    ...
+```
+
+## Example: Filter by stage name(s)
```

Comment:
Can this be moved up, preferably to the top or at least right below the `targets` example? Also, could you add some context for how this can be used inside a stage, or is that better left for a user guide doc?

Reply:
> Also, could you add some context for how this can be used inside a stage, or is that better left for a user guide doc?

I included a minimal example on the very first version of the P.R. but it felt too much for API Reference.

Reply:
Sure, we can have an ad-hoc order for examples. Moved all the way up (and ordered the remaining ones to match the parameter order).

> could you add some context for how this can be used inside a stage, or is that better left for a user guide doc?

Who is "you" here? 😬 Let's put it here as we don't have any guides with API usage and probably won't soon(?)

> included a minimal example

Again, no need to make things fully reproducible. I think @dberenbaum means some more sample code that explains what you would do with the params dict -- in this case set the model training method params I suppose?

Reply:
> more sample code that explains what you would do with the params dict

I tried doing this in 2a70b7dcf. PTAL @daavoo @dberenbaum 

In review app: https://dvc-org-api-get-params-2lnv5kl.herokuapp.com/doc/api-reference/params_show#example-filter-by-stage-names